.github/workflows/pr.yml

```diff
@@ -354,6 +354,13 @@ jobs:
         run: |
           find -name Cargo.lock -type f -exec rm {} \;
 
+
+      # We need `cargo prove` to verify the locked version of `sp1-zkvm`
+      - name: Install SP1
+        run: |
+          cd crates/cli
+          cargo install --locked --force --path .
+
```

Comment:
What are you using this for?

To test the issue in the PR you basically need to run `cargo prove build` on an older version than the `sp1-zkvm` version in a crate.

Or, you run `cargo prove build`, and there have been changes to the syscalls/linking that is non-backwards compatible across versions (e.g. changing the definition of `sys-rand`).

The user seems to have gotten this backwards (unless there was a modification to the arguments for `sys_rand`).